apps/events/migrations/0011_no_activity_pings_to_events_20240619_1436.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.2.11 on 2024-06-19 14:36
+
+from django.db import migrations
+
+from apps.events.models import EventActionType, EventLogStatusChoices
+
+
+def convert_no_activity_pings_to_event(apps, schema_editor):
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    NoActivityMessageConfig = apps.get_model("experiments", "NoActivityMessageConfig")
+    TimeoutTrigger = apps.get_model("events", "TimeoutTrigger")
+    EventAction = apps.get_model("events", "EventAction")
+    EventLog = apps.get_model("events", "EventLog")
+
+    timeout_trigger_content_type = ContentType.objects.get_for_model(TimeoutTrigger)
+
+    for no_activity_ping in NoActivityMessageConfig.objects.all():
+        for experiment in no_activity_ping.experiment_set.all():
+            action = EventAction.objects.create(
+                action_type=EventActionType.SEND_MESSAGE_TO_BOT,
+                params={"message_to_bot": no_activity_ping.message_for_bot}
+            )
+            trigger = TimeoutTrigger.objects.create(
+                action=action,
+                experiment=experiment,
+                delay=no_activity_ping.ping_after * 60,
+                total_num_triggers=no_activity_ping.max_pings,
+            )
+            for session in experiment.sessions.all():
```

Comment:
I don't know where my previous comment went. Anycase, how about filter for sessions where `no_activity_ping_count` > 0? :see_no_evil:

Reply:
OK! I have no idea how many people use OCS... so I wasn't sure how worried about this I should be 😆